packages/babel-helpers/src/helpers.js

```diff
@@ -428,10 +428,18 @@ helpers.objectDestructuringEmpty = template(`
 helpers.objectWithoutProperties = template(`
   (function (obj, keys) {
```

Comment:
Probably better to name them `source` and `excluded`, to match the names in the `CopyDataProperties` algorithm.